### PR TITLE
global: maximum workflow version to 2.0.0

### PR DESCRIPTION
**Pull request by dset0x**
Signed-off-by: Dimitrios Semitsoglou-Tsiapos <dsemitso@cern.ch>

--

This is in preparation of the new workflow module. #2628 shall change this to `>=2.0.0`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def run(self):
     "SQLAlchemy-Utils[encrypted]>=0.30.1",
     "toposort>=1.4",
     "unidecode>=0.04.1",
-    "workflow>=1.2.0",
+    "workflow>=1.2.0<2.0.0",
```
Comment (jirikuncar) [inline setup.py]: IMHO you are missing `,` comma

**Comment by jirikuncar** [comment]
Probably we should merge this to maint-2.0 too.

File: setup.py (hunk 1 above)
Comment (dset0x) [inline setup.py]: Sorry about that and thanks, amended.

**Comment by jirikuncar** [comment]
@dset0x please rebase on top of `maint-2.0` and use `installation` instead of `global`. Thanks

**Comment by jirikuncar** [comment]
@dset0x next time create an issue or at least ask on Gitter. Thanks

**Pull request merged**